lua/mason-registry/init.lua

```diff
@@ -193,4 +193,23 @@ function M.refresh(cb)
     end
 end
 
+---table<string, string[]>
+local aliases = {}
```

Comment:
Happy to implement this however you'd like, I figured it might be beneficial from an API perspective for the actual mapping to be private

Reply:
Looks good to me!